index.html

```diff
@@ -1,271 +1,313 @@
+        </div>
+      </div>
+      <div class="ab-lang card-flex">
+        <img class="ab-img" src="images/Icons1.svg" alt="side-image" />
+        <h2 class="ab-title new">Languages</h2>
+        <ul class="list flex-tool ab-li p-lang">
+          <li>
+            <button class="btn-btn ab-btn" type="button">Javascript</button>
           </li>
+          <li><button class="btn-btn ab-btn" type="button">Ruby</button></li>
+          <li><button class="btn-btn ab-btn" type="button">Html</button></li>
+          <li><button class="btn-btn ab-btn" type="button">Css</button></li>
         </ul>
       </div>
-      <div class="headline-summary">
-        <ul class="list">
+      <div class="ab-lang card-flex">
+        <img class="ab-img" src="images/lang2.svg" alt="side-image" />
+        <h2 class="ab-title new">Frameworks</h2>
+        <ul class="list flex-tool ab-li p-lang">
+          <li>
+            <button class="btn-btn ab-btn" type="button">Bootstrap</button>
+          </li>
+          <li>
+            <button class="btn-btn ab-btn" type="button">Ruby on rails</button>
+          </li>
+          <li><button class="btn-btn ab-btn" type="button">RSpec</button></li>
           <li>
-            I can help you build a product , feature or website Look through
-            some of my work and experience! If you like what you see and have
-            a project you need coded, don’t hesitate to contact me.
+            <button class="btn-btn ab-btn" type="button">Capybara</button>
+          </li>
+          <li>
+            <button class="btn-btn ab-btn" type="button">Selenium</button>
           </li>
         </ul>
       </div>
-      <div>
-        <ul class="social-links list">
+      <div class="ab-lang card-flex">
+        <img class="ab-img" src="images/lang1.svg" alt="side-image" />
+        <h2 class="ab-title new">Skills</h2>
+        <ul class="list flex-tool ab-li p-lang">
+          <li>
+            <button class="btn-btn ab-btn" type="button">codekit</button>
+          </li>
+          <li>
+            <button class="btn-btn ab-btn" type="button">Github</button>
+          </li>
           <li>
-            <a href="https://github.com/admirerbrown/portfolio"><i class="fa-brands fa-github"></i></a>
+            <button class="btn-btn ab-btn" type="button">codepen</button>
           </li>
           <li>
-            <a href="http://www.linkedin.com/in/samuel-ntow-kyere-5036741b4"><i class="fa-brands fa-linkedin"></i></a>
+            <button class="btn-btn ab-btn" type="button">Gitlab</button>
           </li>
           <li>
-            <a href="https://twitter.com/brown_admirer"><i class="fa-brands fa-twitter"></i></a>
+            <button class="btn-btn ab-btn" type="button">Terminal</button>
           </li>
         </ul>
       </div>
-      <div class="headline-button">
-        <img src="images/Action.svg" />
+    </section>
+
+    <!-- contact me section -->
+    <section id="abt-contact">
```

Comment:
- [ ] Great job styling your form and contact section. However, according to the project requirements, you are supposed to make use of the CSS flexbox to align the items of this section. Kindly do so.

![image](https://user-images.githubusercontent.com/86318284/199233335-670f632a-3199-4aa2-9e0e-3713405b8fb7.png)